test/conformance/parallel/ptptestconfig_test.go

```diff
@@ -0,0 +1,27 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfiguration(t *testing.T) {
+	cfg := PtpTestConfig{}
+	path, _ := os.Getwd()
+	loadPtpTestConfig(fmt.Sprintf("%s/%s", path, "testdata/cfg1.yaml"), &cfg)
```

Comment:
how does this work when you use docker run ?
testdata/cfg1.yaml, will we have cfg2.yaml, cfg3.yaml , if yes, why ?
If there is one global configuration to override then we should have a one config with meaningful name for it .

Reply:
I think I did not understand your comments. How are other unit tests running from docker? `go test` from the right directory works.

Reply:
The config files inside the `testdata`directory are for unit tests. The config file that contains test configurations is ptpconfig.yaml